go.sum

```diff
@@ -1,5 +1,7 @@
 github.com/Harvey-OS/ninep v0.0.0-20190302184144-8010cab8eac9 h1:vcT+syCMklD8M/k/PcNYehmCRYrp8cyO7pjlZC1/vwg=
 github.com/Harvey-OS/ninep v0.0.0-20190302184144-8010cab8eac9/go.mod h1:YmLTajv/R+cjEsJ8/vfGOckeEBbz06tgHlYBSpbe+eo=
+github.com/Harvey-OS/ninep v0.0.0-20200701230449-5148a8f50134 h1:0Qo7JlCQOUMhO2+L9Mtujhz/cWykNG9Mmx5goPGLCLA=
```

Comment:
weren't we supposed to .gitignore thesse?

Reply:
go.* removed from PR.

Reply:
The `go.mod` and `go.sum` files are generally checked into git. From https://blog.golang.org/using-go-modules:
> The go command uses the go.sum file to ensure that future downloads of these modules retrieve the same bits as the first download, to ensure the modules your project depends on do not change unexpectedly, whether for malicious, accidental, or other reasons. Both go.mod and go.sum should be checked into version control.